source/cpu/cpu.go

```diff
@@ -19,14 +19,11 @@ package cpu
 import (
 	"io/ioutil"
 	"log"
-	"path"
 
 	"sigs.k8s.io/node-feature-discovery/source"
 )
 
-const (
-	cpuDevicesBaseDir = "/sys/bus/cpu/devices"
-)
+const ()
```

Comment:
Can be removed I think

Reply:
Thx, will be dropped! I really thought I removed that 🤔

Reply:
Fixed